tracer/src/Datadog.Trace/Iast/IastModule.cs

```diff
@@ -865,6 +866,16 @@ internal static void OnEmailHtmlInjection(object? message)
         GetScope(messageDuck.Body, IntegrationId.EmailHtmlInjection, VulnerabilityTypeName.EmailHtmlInjection, OperationNameEmailHtmlInjection, taintValidator: Always, safeSources: _dbSources, exclusionSecureMarks: SecureMarks.Xss);
     }
 
+    public static void LogAspectException(Exception ex, string message)
+    {
+        if (!LoggedAspectExceptionMessages.Add(message.GetHashCode()))
+        {
+            return;
```

Comment:
Here we could trace as debug level

Reply:
This function should be try catched also

Reply:
Changed in e609bbb and eb9d9b8 😄